### PR TITLE
feat: Add Connext Decoders

### PR DESCRIPTION
**Pull request by karanpargal**

### CODE REVIEW

File: services/decoder/protocols/connext/connext.decoders.ts [hunk 1]
```diff
@@ -0,0 +1,893 @@
...
+            {
+                heading: "Key",
+                value: decoded.key,
+                type: "text",
+            },
+            {
+                heading: "Amount",
+                value: decoded.amount.toString(),
+                type: "text",
+            },
```
Comment (jagnani73) [inline services/decoder/protocols/connext/connext.decoders.ts:64]: Should this not be a token?

File: services/decoder/protocols/connext/connext.decoders.ts [hunk 2]
```diff
@@ -0,0 +1,893 @@
...
```
Comment (jagnani73) [inline services/decoder/protocols/connext/connext.decoders.ts:471]: Token?

File: services/decoder/protocols/connext/constants/domain-mapping.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+    6648936: "Ethereum Mainnet",
+    1886350457: "Polygon",
+    1869640809: "Optimism",
+    1634886255: "Arbitrum One",
+    6778479: "Gnosis Chain",
+    6450786: "BNB Chain",
+    1818848877: "Linea",
+    1835365481: "Metis",
+    1650553709: "Base",
+};
```
Comment (jagnani73) [inline services/decoder/protocols/connext/constants/domain-mapping.ts:11]: No need for a separate file, create this in the decoder file itself

**Review by jagnani73 — CHANGES_REQUESTED** [review]

File: services/decoder/protocols/connext/constants/domain-mapping.ts (hunk 3 above)
Comment (karanpargal) [reply services/decoder/protocols/connext/constants/domain-mapping.ts]: Resolved in: 44b5616f9a2fc157809c3ca616cdcfa3d50ef712

**Review by jagnani73 — APPROVED** [review]

**Review by jaypozo — APPROVED** [review]

**Pull request merged**